docs/semgrep-assistant/overview.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: overview
+append_help_link: true
+title: AI-assisted triage and autofix
+hide_title: true
+description: Learn how Semgrep Assistant can provide recommendations for triage and remediation of Semgrep findings. 
+tags:
+  - Semgrep Code
+  - Semgrep Supply Chain
+  - Semgrep Secrets
+  - Semgrep Cloud Platform
+  - Semgrep Assistant
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Semgrep Assistant overview
+
+Semgrep Assistant provides GPT-4-powered security recommendations to help you review, triage, and remediate your Semgrep findings.
+
+![Screenshot of Semgrep Assistant message in GitHub](/img/semgrep-assistant-github.png)
+*Figure*. Semgrep Assistant detects a false positive.
+
+## Support and availability
+
+Semgrep Assistant supports findings generated by Semgrep Code, Semgrep Supply Chain, and Semgrep Secrets, and, in many cases, requires using Semgrep Cloud Platform. It is not available on the Semgrep CLI.
```

Comment:
```suggestion
Semgrep Assistant supports findings generated by Semgrep Code, Semgrep Supply Chain, and Semgrep Secrets, and, in many cases, requires using Semgrep Cloud Platform. It is not available on the Semgrep CLI.
```

I'm not sure Semgrep Assistant can assist with SSC or Secrets findings - for example, there are no component tags in the Supply Chain or Secrets page.

Can we confirm:

- Component tags - Code only?
- Rules recommendations in Dashboard - Code only?
- Auto-triage - Findings page, so Code only?
- Autofix - Code only
- Rule-writing assistance - we do let users write their own Secrets rules, but not sure if the playground supports this; I am aware this is for Code, because we don't generally enable user-written custom SSC rules.

Maybe priority inbox is a feature that covers all 3 products?

I think verifying this info will result in some minor changes to this doc (headings are arranged by product then feature?)

Reply:
Assistant works across all three products; see [the discussion beginning at 19:25](https://us-30862.app.gong.io/call?id=5043540070866739060).

As it is, I think all of those things that you've flagged as Code-only are written as such; I will add docs for SSC and Secrets as they roll out in the UI. That said, Priority inbox does already cover all products.

The only thing I'm unsure about is secrets rule writing; I'll check on this.